### DISCUSSION TITLE
How do I get completion in JSX files (with LSP)?

### DISCUSSION THREAD

**Discussion by kujohn** [company-mode/company-mode#1160] [Q&A] [answered]
How does one modify company to allow auto complete in rjsx-mode for javascript/js?

**Comment by kujohn** [comment, marked as answer]
nevermind... i think reinstalling and setting the right lsp did the work?

**Reply by dgutov** [reply]
Probably.

The settings you need to have it all work are really out of scope for this project.

lsp-mode guides should be a better help, e.g. https://emacs-lsp.github.io/lsp-mode/page/lsp-typescript/.
